framework/source/class/qx/ui/core/Blocker.js

```diff
@@ -263,13 +263,19 @@ qx.Class.define("qx.ui.core.Blocker",
      */
     _backupActiveWidget : function()
     {
-      var focusHandler = qx.event.Registration.getManager(window).getHandler(qx.event.handler.Focus);
+      var focusHandler = qx.event.Registration.getManager(window).getHandler(qx.event.handler.Focus),
+        activeWidget = qx.ui.core.Widget.getWidgetByElement(focusHandler.getActive()),
+        focusedWidget = qx.ui.core.Widget.getWidgetByElement(focusHandler.getFocus());
 
-      this.__activeElements.push(qx.ui.core.Widget.getWidgetByElement(focusHandler.getActive()));
-      this.__focusElements.push(qx.ui.core.Widget.getWidgetByElement(focusHandler.getFocus()));
+      this.__activeElements.push(activeWidget);
+      this.__focusElements.push(focusedWidget);
 
-      if (this._widget.isFocusable()) {
-        this._widget.focus();
```

Comment:
wouldn't this also automatically cause any other widget to blur?

Reply:
The problem is that when blocker is attached to a composite, this piece of code is not doing anything, because composite is not focusable.